src/main/kotlin/no/nav/amt/person/service/kafka/ingestor/EndretDeltakerIngestor.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.amt.person.service.kafka.ingestor
+
+import no.nav.amt.person.service.kafka.ingestor.dto.DeltakerDto
+import no.nav.amt.person.service.nav_bruker.NavBrukerService
+import no.nav.amt.person.service.utils.JsonUtils.fromJsonString
+import org.springframework.stereotype.Service
+
+@Service
+class EndretDeltakerIngestor(
+	private val navBrukerService: NavBrukerService
+) {
+	fun ingest(value: String) {
+		val deltakerRecord = fromJsonString<DeltakerDto>(value)
+		navBrukerService.oppdaterKontaktinformasjon(
+			personident = deltakerRecord.personalia.personident,
+			deltakerId = deltakerRecord.id
```

Comment:
Bare nit, men hva om vi sjekker om bruker finnes før vi kaller oppdaterKontakinformasjon og sender hele brukeren som et parameter i stedet?

Jeg synes det er litt ryddigere at vi ikke har noe forhold til deltakerId lengre inn i applikasjonen hvis vi ikke må. Også blir funksjonen mer gjenbrukbar hvis det trengs senere.

Reply:
ja, det bør gå an :)

Reply:
da kan jeg kanskje like godt gjøre sjekken på når den sist ble synkronisert her ute også?

Reply:
hmm, men da blir koden vanskeligere å teste da, så kanskje jeg lar den sjekken være der den er..

Reply:
Der nevner du noe 😅 Jeg tror vi bør ha en integrasjonstest for denne ingestoren.

Reply:
ja, hvis jeg flytter den delen av logikken må vi teste ingestoren. Det er bare så mas å lage tester for ingestorene, tar en evighet å spinne opp..

Reply:
det ble faktisk ikke så ille 😅